packages/cli/src/commands/doctor/healthchecks/__tests__/androidHomeEnvVariable.test.ts

```diff
@@ -0,0 +1,56 @@
+import androidHomeEnvVariables from '../androidHomeEnvVariable';
+import getEnvironmentInfo from '../../../../tools/envinfo';
+import {EnvironmentInfo} from '../../types';
+import {Ora} from 'ora';
+
+jest.mock('../common');
+
+import {logManualInstallation} from '../common';
+
+describe('androidHomeEnvVariables', () => {
+  const OLD_ENV = process.env;
+
+  it('returns a message if no ANDROID_HOME is defined', async () => {
+    delete process.env.ANDROID_HOME;
```

Comment:
I was wondering if this can cause problems but I saw the `afterEach` so it seems good to go 🙂

Reply:
OLD_ENV will contain the environment before running any test, so since we restore it on afterEach, yeah it should be good to go 😉